lib/godwoken_explorer/account/current_udt_balance.ex

```diff
@@ -82,6 +80,7 @@ defmodule GodwokenExplorer.Account.CurrentUDTBalance do
 
     (bridged_udt_balances ++ udt_balances)
     |> Enum.sort_by(&Map.fetch(&1, :updated_at))
```

Comment:
it seems
`|> Enum.sort_by(&Map.fetch(&1, :updated_at))`
can use
`|> Enum.sort_by(&Map.fetch(&1, :updated_at), :desc)`

replace